routes/web.php

```diff
@@ -40,8 +42,10 @@
         }
     }
 
-    foreach (UserResource::router()->routes as $route) {
-        Route::get(UserResource::getSlug() . '/' . $route->uri, $route->page)
-            ->name(UserResource::getSlug() . '.' . $route->name);
-    }
+    Route::middleware(AuthorizeAdmins::class)->group(function () {
+        foreach (UserResource::router()->routes as $route) {
```

Comment:
Shouldn't this be using `Filament::userResource()` instead?

Reply:
Yeah, corrected.